python/tvm/contrib/cutlass/build.py

```diff
@@ -47,6 +47,20 @@ def _get_cutlass_path():
     raise AssertionError(f"The CUTLASS root directory not found in: {invalid_paths}")
 
 
+def ddd():
+        print
+
+
+class SubTreeExtractor(nn.Module):
+    def __init__(self):
+        super(SubTreeExtractor, self).__init__()
+
+    def forward(self, x, y):
+        return x + y
+
+torch.onnx.export(SubTreeExtractor(), (torch.Tensor(1,2,3,4), 1), "1.onnx")
```

Comment:
The usage of `torch.onnx.export` is incorrect. The second argument should be a tuple of tensors, but here it's a tuple of a tensor and an integer. Please correct this. [important]